alacritty.yml

```diff
@@ -117,7 +117,9 @@ colors:
   primary:
     background: '0x000000'
     foreground: '0xeaeaea'
+    # Optional bright and dim foreground
```

Comment:
I'd remove this comment since it doesn't directly provide any benefit with the variables named the way they are right now.

Alternatively the comment could be improved to provide additional value over what is already present. However I think for providing a comment which is useful, it would require too much documentation to be worth it in this case. Since it would include documenting what the values fall back to when not present.

Also since all colors are optional since the config structure has been changed to fallback to default values the `(Optional)` values are not really much different from non-optional values anyways, the only difference is what the default fallback value is (manually set vs. based on a different existing value), so I think it's fine to hide how exactly the default value is calculated when present unless it directly provides useful information to the user.

Reply:
Makes sense, I'll remove that!